CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Changelog
 
+## [1.56](https://github.com/brave/brave-ios/releases/tag/v1.56)
+
+### Web3
+
+- Implemented “Select a token to send” view which allows selecting an account and an asset at the same time. ([#7523](https://github.com/brave/brave-ios/issues/7523))
+- Removed network name from the “Public Encryption Key Request” panel. ([#7495](https://github.com/brave/brave-ios/issues/7495))
+- Improved NFT importing in Brave Wallet by introducing a new button to add custom assets quicker in NFT tab view and also simplifying the “Add Custom Asset” form by moving moving “Token address” field to the top. ([#7556](https://github.com/brave/brave-ios/issues/7556))
```

Comment:
Maybe shorten it to 
`Updated NFT tab view with new "Add Custom Asset" button and ability to hide NFTs via long press"